easybuild/easyconfigs/a/ABINIT/ABINIT-9.6.2-foss-2022a.eb

```diff
@@ -0,0 +1,75 @@
+easyblock = 'ConfigureMake'
+
+name = 'ABINIT'
+version = '9.6.2'
+
+homepage = 'https://www.abinit.org/'
+description = """
+ABINIT is a package whose main program allows one to find the total energy, charge density and electronic structure of
+systems made of electrons and nuclei (molecules and periodic solids) within Density Functional Theory (DFT), using
+pseudopotentials and a planewave or wavelet basis.
+"""
+
+toolchain = {'name': 'foss', 'version': '2022a'}
+toolchainopts = {'usempi': True, 'openmp': True, 'pic': True}
+
+source_urls = ['https://www.abinit.org/sites/default/files/packages/']
+sources = [SOURCELOWER_TAR_GZ]
+checksums = ['b018c2ff24338a5952c5550a7e09d4c7793b62402c7aa4e09273e9a666e674fb']
+
+builddependencies = [
+    ('Python', '3.10.4'),
+]
+dependencies = [
+    ('libxc', '5.2.3'),
+    ('netCDF', '4.9.0'),
+    ('netCDF-Fortran', '4.6.0'),
+    ('HDF5', '1.12.2'),
+    ('Wannier90', '3.1.0'),
+]
+
+# Needed due to changes in GCC10.
+configopts = 'FCFLAGS="-fallow-argument-mismatch -ffree-line-length-none $FCFLAGS" '
+configopts += 'FFLAGS="-fallow-argument-mismatch $FFLAGS" '
+
+# Ensure MPI
+configopts += '--with-mpi="yes" '
+# configopts += ' FC="mpiifort" CC="mpiicc" CXX="mpiicpc" '
+
+# Enable OpenMP
+configopts += '--enable-openmp="yes" '
+
+# BLAS/Lapack from OpenBLAS
+configopts += '--with-linalg-flavor="openblas"  LINALG_LIBS="${LIBLAPACK_MT}" '
```

Comment:
FlexiBLAS?

Reply:
Good point. This was actually linked correctly, since the framework will set `LIBLAPACK_MT` to `-lflexiblas -lgfortran`. I improved it in https://github.com/easybuilders/easybuild-easyconfigs/commit/a97bd9612cf85846260af2af0bbbbbea776e83c6